### PR TITLE
Feat/enable transfers

### PR DESCRIPTION
**Pull request by EnoF**

### CODE REVIEW

File: spirekey/src/components/SendForm/SendForm.tsx [hunk 1]
```diff
@@ -1,150 +1,323 @@
...
+          withCap(`coin.TRANSFER`, account.accountName, receiver, {
+            decimal: amount.toFixed(8),
+          }),
+          withCap(`coin.GAS`),
+        ],
+      ),
+    ),
+  );
+  return tx;
+};
```
Comment (eileenmguo) [inline spirekey/src/components/SendForm/SendForm.tsx:90]: why are you adding all devices for an account?

File: spirekey/src/components/SendForm/SendForm.tsx [hunk 2]
```diff
@@ -1,150 +1,323 @@
...
+            ],
+          },
+        ],
+      );
+      await isReady();
+      await Promise.all(
+        transactions.map(async (tx) => {
+          console.warn("DEBUGPRINT[5]: SendForm.tsx:170: tx=", tx)
+          const res = await l1Client.local(tx);
+          console.log('Preflight result', res);
```
Comment (eileenmguo) [inline spirekey/src/components/SendForm/SendForm.tsx:172]: did you intend to leave logs?

File: spirekey/src/components/SendForm/SendForm.tsx [hunk 3]
```diff
@@ -1,150 +1,323 @@
...
+    max: parseFloat(account.balance),
+  });
 
-  if (!account)
+  if (tx)
     return (
-      <div>
-        Account not found <ButtonLink href="/">Go back home</ButtonLink>
-      </div>
+      <CardContainer hasPadding>
```
Comment (eileenmguo) [inline spirekey/src/components/SendForm/SendForm.tsx:195]: I would suggest already using the updated card pattern components from @kadena/kode-ui and SpireKeyCardContentBlock (already has the icon preconfigured). I'm almost done converting the rest of them, there's just one styling issue with the title size on the following content blocks when they are stacked

**Review by eileenmguo — APPROVED** [review]

File: spirekey/src/app/(wallet)/accounts/[caccount]/devices/[cid]/send/page.tsx [hunk 4]
```diff
@@ -1,18 +1,11 @@
...
 });
 
 export default function SendPage() {
-  return (
-    <Box margin="lg">
-      <h1>Send</h1>
-      <SendForm />
-    </Box>
-  );
+  return <SendForm />;
```
Comment (Randynamic) [inline spirekey/src/app/(wallet)/accounts/[caccount]/devices/[cid]/send/page.tsx:10]: Can't we move this logic to the higher component as it's not doing a lot anymore?

**Review by Randynamic — APPROVED** [review]

**Review by Randynamic — APPROVED** [review]

**Pull request merged**